src/nodes.coffee

```diff
@@ -4048,6 +4097,10 @@ exports.Op = class Op extends Base
   toString: (idt) ->
     super idt, @constructor.name + ' ' + @operator
 
+  ast: (o) ->
+    @checkContinuation o if @isYield() or @isAwait()
```

Comment:
This `Op` update isn't related to `YieldReturn`/`AwaitReturn`, it just allows us to test for "simple" `yield`/`await` outside of a function throwing an error when generating AST (in addition to the comparable tests for `yield return`/`await return` that I was adding)